trace/cloud-trace-demo-app/app/demo-service-a.yaml

```diff
@@ -0,0 +1,40 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: cloud-trace-demo-a
+  labels:
+    app: cloud-trace-demo-app-a
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: cloud-trace-demo-app-a
+  template:
+    metadata:
+      name: cloud-trace-demo-a
+      labels:
+        app: cloud-trace-demo-app-a
+    spec:
+      containers:
+      - name: cloud-trace-demo-container
+        image: gcr.io/cloud-trace-demo/demo-image:latest
+        command:
+        - python
+        args:
+        - app.py
+        - --keyword=Hello
```

Comment:
nit: I'm not sure if it's this hello or another one, but whichever Hello gets printed in response with the cURL command should have a newline character

Reply:
Added a new line character on the last service, service c's yaml template.